app/presenters/document_export_presenter.rb

```diff
@@ -143,21 +140,18 @@ def present_translations(edition)
   end
 
   def present_unpublishing(edition)
-    if edition.unpublishing
-      edition
-        .unpublishing
-        .as_json(
-          only: %i[
-            id
-            explanation
-            alternative_url
-            redirect
-            created_at
-            updated_at
-          ],
-        )
-        .merge(unpublishing_reason: edition.unpublishing.unpublishing_reason.name)
-    end
+    return {} unless edition.unpublishing
```

Comment:
I talked to @kevindew before doing this, and he suggested returning `nil` rather than an empty object as that is the current pattern. e.g. `government` returns `nil` if the document isn't assigned to a government.

Reply:
Very true, didn't see that example! Instead I've changed the associations returning an empty hash to `nil` so now the default for everything is either an empty array or nil! 🎉